articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+
+7. In the local gateway blade, under *Settings*, select **Configuration**.
+
+    ![Azure Public local network gateway](images/azs-public-browser-local-gateway-configuration-button.png)
+
+8. Change the **IP address** field to the IP taken from the Azure Stack connection.
+
+    ![Azure Public local gateway public IP address](images/azs-public-browser-local-gateway-ip.png)
+
+9. Click **Save**.
+
+### Update the local Gateway IPs in Azure Stack
+
+1. In the Public Azure portal, navigate to the Connections blade by clicking **All services** in the favourites panel, then selecting **Connections** under the *networking* section.
+
+    ![Connections button under all services in public Azure](images/azs-public-browser-services-connections-button.png)
+
+2. Select the VPN connection you created in the earlier stages.
+
+3. Make note of the public IP this can be found in the *Virtual network gateway* section.
+
+    ![Public IP of virtual network gateway in Public Azure](images/azs-public-browser-connection-ip.png)
+
+4. Log in to the [Azure Stack portal](https://portal.frn00006.azure.ukcloud.com).
+
+5. In the Azure Stack portal, navigate to the *Local Network Gateway* blade by clicking **All services** in the favourites panel, then selecting **Local network gateways** under the *networking* section.
+
+    ![Azure Stack portal local network gateway](images/azs-browser-local-network-gateway-all-services.png)
+
+6. Select the local network gateway that you created in the earlier steps.
+
+7. In the *local gateway* blade, under *Settings*, select **Configuration**.
+
+    ![Azure Stack local network gateway configuration](images/azs-browser-local-network-gateway-configuration.png)
+
+8. Change the *IP address* field to the IP taken from the Public Azure connection.
+
+    ![Azure Stack local network gateway IP address](images/azs-browser-local-network-gateway-ip-address.png)
+
+9. Click **Save**.
+
+## Verify the VPN connection
+
+After configuring the VPN device on your local network, you can verify the VPN connection with the following steps:
+
+1. Log in to the [Azure Stack portal](https://portal.frn00006.azure.ukcloud.com).
+
+2. Navigate to your virtual network gateway by clicking **All services** in the favourites panel, then selecting **Virtual network gateways** under the networking section.
+
+    ![Azure Stack virtual network gateways](images/azs-browser-virtual-network-gateway-all-services.png)
+
+3. Select your virtual network gateway from the list.
+
+4. In the blade for your virtual network gateway, select **Connections** under the **Settings** section.
+
+    ![Azure Stack virtual network gateway connections](images/azs-browser-virtual-network-gateway-connections.png)
+
+5. Select your connection from the list.
+
+6. In the *new* blade, you can view more information about your connection. The connection status will be **Connected** if there is a successful connection.
```

Comment:
In the blade for your connection, you can view more information. The connection status will be **Connected** if there is a successful connection.